.gitignore

```diff
@@ -20,3 +20,5 @@ resources
 !**/.yarn/sdks
 !**/.yarn/unplugged
 !**/.yarn/versions
+
+.pnp
```

Comment:
Not sure where this is created or why, but it sounds like we need it or it generates by error on your side. Either way I think it needs to be removed.

Reply:
Ups, sorry, I have not noticed this comment. You mean it should not be ignored?

Reply:
I have force-pushed without this change to the master

Reply:
Yeah, I don't think it should be ignored, PNP stands for plug and play which is a Yarn feature that replaces node_modules, we also use another Yarn feature called zero install that vendors all the dependencies. The end result is, if you checkout the repository you don't need to fetch anything to build. For example, we don't need `yarn install` on the CI:

https://github.com/apache/camel-website/blob/33fedc838299b8b0b84aa24726ff813eb7ec5218/Jenkinsfile#L58

In that way it's odd that you get this file/directory created, as we also have a GitHub action that checks that everything is present in `.yarn` and in sync with `.pnp.js` and `yarn.lock`:

https://github.com/apache/camel-website/blob/33fedc838299b8b0b84aa24726ff813eb7ec5218/.github/workflows/pr.yaml#L28

Nevermind, I'll remove that from .gitignore.

Reply:
> I'll remove that from .gitignore.

As I said, the change is not in the master anymore.